Dockerfile

```diff
@@ -2,22 +2,11 @@
 FROM ubuntu:latest
 MAINTAINER Greg Castle github@mailgreg.com
 
-RUN mkdir -p /usr/share/grr/scripts
-COPY scripts/install_script_ubuntu.sh /usr/share/grr/scripts/install_script_ubuntu.sh
-ENV UPGRADE=false
-# Install our dependencies, use latest requirements.txt
-RUN bash /usr/share/grr/scripts/install_script_ubuntu.sh -dy -r https://raw.githubusercontent.com/google/grr/master/requirements.txt
-
-# Download the client templates now to get better caching from Docker.
```

Comment:
I don't think we want to do this. The nice thing about the docker image is that once you have the requirements installed and the templates downloaded those are cached by docker. If we do it this way you will need to re-run those entire installs any time any of the source files are modified. This is significantly slower. Lets keep these bits as is and do the necessary skipping inside the install_server_from_src script.